en/deploy/README.md

```diff
@@ -245,7 +245,7 @@ You can also go to the "Files" page and navigate around using PythonAnywhere's b
 Your site should now be live on the public Internet!  Click through to the PythonAnywhere "Web" page to get a link to it. You can share this with anyone you want. :)
 
 
-> **Note** This is a beginners' tutorial, and in deploying this site we've taken a few shortcuts which aren't ideal from a security point of view.  If and when you decide to build on this project, or start a new project, you should review the [Django deployment checklist](https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/) for some tips on securing your site.
+> **Note** This is a beginners' tutorial, and in deploying this site we've taken a few shortcuts which aren't ideal from a security point of view.  If and when you decide to build on this project, or start a new project, you should review the [Django deployment checklist](https://docs.djangoproject.com/en/3.2/howto/deployment/checklist/) for some tips on securing your site.
```

Comment:
Should we introduce a variable like `django_minor`, `django_doc_version` or something like that?

Reply:
I think we could, though is there any gotchas we should consider for this templating system where the variable could not be replaced? (e.g. related to #1694?)

Maybe this is something we can try to tackle in a separate PR. Django docs are also available in other languages, so playing around to replace the `/en/` in the docs URL could be useful too.

Reply:
Sounds ok to me.